sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -82,15 +87,30 @@ public SdkRequestT setNodeAccountIds(List<AccountId> nodeAccountIds) {
 
     @FunctionalExecutable
     public CompletableFuture<O> executeAsync(Client client) {
-        return onExecuteAsync(client).thenCompose((v) -> executeAsync(client, 1, null));
+        return onExecuteAsync(client).thenCompose((v) -> {
+            if(nodeAccountIds.isEmpty()) {
+                throw new IllegalStateException("Request node account IDs were not set before executing");
+            }
+            if(nodes.isEmpty()) {
+                for(AccountId id : nodeAccountIds) {
+                    @Nullable
+                    Node node = client.network.networkNodes.get(id);
```

Comment:
As above, can we use `var node` here instead of declaring the type for SDK consistency.